src/main/scala/temple/generate/server/go/service/GoServiceGenerator.scala

```diff
@@ -49,20 +44,29 @@ object GoServiceGenerator extends ServiceGenerator {
       File(root.kebabName, s"${root.kebabName}.go") -> mkCode.doubleLines(
         GoCommonGenerator.generatePackage("main"),
         GoServiceMainGenerator
-          .generateImports(root, usesBase64, usesTime, usesComms, usesMetrics, clientAttributes, operations),
+          .generateImports(root, usesBase64, usesTime, usesComms, usesMetrics, root.requestAttributes, root.operations),
```

Comment:
perhaps could do with refactoring now that requestAttributes and operations are defined on the root, so only the root and the bools are passed